harness/determined/cli/render.py

```diff
@@ -292,3 +305,31 @@ def print_json(data: Union[str, Any]) -> None:
 def report_job_launched(_type: str, _id: str) -> None:
     msg = f"Launched {_type} (id: {_id})."
     print(termcolor.colored(msg, "green"))
+
+
+def model_to_json(model: Model) -> Dict[str, Any]:
```

Comment:
should these be private?

Reply:
These need to be accessible to the CLI. ATM, I don't think we have any way of saying "OK for CLI, bad for anyone else." We have some notion of "private to within determined/harness" that we're not consistent about, but that we've got tickets out to make consistent. The CLI is one of the things that the ticket is going to have to address.

So... yes/it's complicated, and also not here in this PR.